crates/jstzd/tests/jstzd_test.rs

```diff
@@ -117,14 +116,113 @@ async fn jstzd_test() {
     })
     .await;
     assert!(node_destroyed);
-
     assert!(!jstzd.baker_healthy().await);
+    let rollup_health = rollup_health_check(&rollup_rpc_endpoint.clone()).await;
+    assert!(rollup_health.is_err());
     assert!(!jstzd.health_check().await);
 
     // stop should be idempotent and thus should be okay after jstzd is already stopped
     jstzd.stop().await.unwrap();
 }
 
+async fn build_configs(
```

Comment:
Note: These are probably the default values we want to instantiate jstzd with. I expect that when we put the cli together, these values would be moved into `src/` instead.

Reply:
Yes. I've implemented that actually just waiting for the prerequisite branches to be merged. The default values will be set when the program parses and processes user config.